### PR TITLE
UIDEXP-57: update mapping profile formatter

### PR DESCRIPTION
**Pull request by Vladyslav-Velytskyi**
### Description

- Update formatter for column `updatedBy` in case it doesn't get any value
- Update formatter for column `folioRecord` in case it get an array of records

### Issue
[UIDEXP-57](https://issues.folio.org/browse/UIDEXP-57)
relates to [PR](https://github.com/folio-org/ui-data-export/pull/70)

### Screenshot
image

### CODE REVIEW

**Comment by VictorSoroka1** [comment]
Please ensure that all quality gates are passed.

File: lib/ListTemplate/listTemplate.js [hunk 1]
```diff
@@ -9,6 +10,10 @@ export const listTemplate = {
     return <FormattedDate value={updatedDate} />;
   },
   updatedBy: record => {
+    if (!record.userInfo) {
```
Comment (VictorSoroka1) [inline lib/ListTemplate/listTemplate.js:13]: Per comment from PO the dash is not needed when the is no data to display.

File: lib/ListTemplate/listTemplate.js [hunk 2]
```diff
@@ -9,6 +10,10 @@ export const listTemplate = {
...
+      return <NoValue />;
```
Comment (VictorSoroka1) [inline lib/ListTemplate/listTemplate.js:14]: Per comment from PO the dash is not needed when the is no data to display.

```suggestion
      return null;
```

File: lib/Settings/MappingProfiles/listTemplate.js [hunk 3]
```diff
@@ -7,8 +7,12 @@ import { listTemplate as defaultListTemplate } from '../../ListTemplate';
 export const listTemplate = {
   ...defaultListTemplate,
   folioRecord: record => {
-    const { existingRecordType } = record;
+    const { recordTypes } = record;
 
-    return <FormattedMessage id={FOLIO_RECORD_TYPES[existingRecordType].captionId} />;
+    return recordTypes.map((recordType, index) => {
```
Comment (VictorSoroka1) [inline lib/Settings/MappingProfiles/listTemplate.js:12]: Let's simplify a bit.
```suggestion
    return recordTypes
      .map((recordType, index) => <FormattedMessage id={FOLIO_RECORD_TYPES[recordType].captionId} />)
      .join(', ')
```

Comment (Vladyslav-Velytskyi) [reply lib/Settings/MappingProfiles/listTemplate.js:12]: I tried this issue, but it returns `[object Object]`

Comment (VictorSoroka1) [reply lib/Settings/MappingProfiles/listTemplate.js:12]: Ah, yes, fair enough. Sounds good then, then make sure to cover with tests this logic. Ideally to end up with 100% percent. For unit test approach which we have there, it should not be a problem.

File: lib/Settings/MappingProfiles/listTemplate.js [hunk 4]
```diff
@@ -7,8 +7,12 @@ import { listTemplate as defaultListTemplate } from '../../ListTemplate';
...
+      const recordTypeValue = <FormattedMessage id={FOLIO_RECORD_TYPES[recordType].captionId} />;
+
+      return !index ? recordTypeValue : (<>, {recordTypeValue}</>);
```
Comment (VictorSoroka1) [inline lib/Settings/MappingProfiles/listTemplate.js:15]:
```suggestion
      return !i ? recordTypeValue : <>, {recordTypeValue}</>;      
```

**Comment by YevheniiMaltsev** [comment]
Please, update CHANGELOG file

**Pull request merged**